keystore/keystore.go

```diff
@@ -81,16 +81,38 @@ type Key struct {
 	Content []byte
 }
 
+// ExportMode constants describe which data to export from key storage.
+type ExportMode int
+
+// ExportMode flags.
+const (
+	// Export only public key data.
+	ExportPublicOnly ExportMode = 0
+	// Export private and public key data.
+	ExportPrivateKeys = (1 << iota)
```

Comment:
hm, what the logic of such assigning?) 0, 1, 4... it's 000, 001, 100 in binary format.. 
if it just moved from acra-backup here and these values weren't dumped somewhere and used only internally, we can use more predictable values for that. Just iota or `1 << iota` if we want use them as bit masks